buckets/local/buckets.go

```diff
@@ -0,0 +1,322 @@
+package local
```

Comment:
This file contains the `Buckets` (plural) object, which is used to create N buckets at different locations.

Reply:
See docstrings for more